packages/create-block-app/create-block-app.js

```diff
@@ -20,7 +20,8 @@ const templatePackageName = "block-template";
     process.exit();
   }
 
-  const blockPath = process.argv[3] ?? blockName;
+  // @todo: replace with `process.argv[3] ?? blockName` after upgrading ESLint
+  const blockPath = process.argv[3] ? process.argv[3] : blockName;
```

Comment:
Getting syntax error when running `yarn lint:eslint`. Might be to do with outdated built-in babel. I’ll update ESLint to fix that, which might be handy before #168 anyway:

```
=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.4.0

YOUR TYPESCRIPT VERSION: 4.5.4

Please only submit bug reports when using the officially supported version.

=============
```

Reply:
Same problem with `} catch {` below